src/pages/api/identities/[id].ts

```diff
@@ -102,23 +169,20 @@ const handler: Handler = (request, response): Promise<void> => {
   });
 };
 
-const wrappedHandler = wrapMiddlewares(
-  [
-    tracingMiddleware(getTracer()),
-    rateLimitingMiddleware(getTracer(), logger, {
-      windowMs: 300000,
-      requestsUntilBlock: 200,
-    }),
-    recoveryMiddleware(getTracer()),
-    sentryMiddleware(getTracer()),
-    errorMiddleware(getTracer()),
-    loggingMiddleware(getTracer(), logger),
-    authMiddleware(getTracer()),
-  ],
-  handler,
-  "/api/identity/delete-identity",
-);
+const middlewares: Middleware<NextApiRequest, NextApiResponse>[] = [
+  tracingMiddleware(getTracer()),
+  rateLimitingMiddleware(getTracer(), logger, {
+    windowMs: 300000,
+    requestsUntilBlock: 200,
+  }),
+  recoveryMiddleware(getTracer()),
+  sentryMiddleware(getTracer()),
+  errorMiddleware(getTracer()),
+  loggingMiddleware(getTracer(), logger),
+  authMiddleware(getTracer()),
+];
 
 export default new Endpoint<NextApiRequest, NextApiResponse>()
-  .delete(wrappedHandler)
+  .get(wrapMiddlewares(middlewares, get, "/api/identities/[id]"))
+  .delete(wrapMiddlewares(middlewares, destroy, "/api/identities/[id]"))
   .getHandler();
```

Comment:
Nice one with the middlewares declaration, I didn't do like that on the address handler, but I should have, it's way more readable and clean. We will need to update that on next PRs 🙂